ibis/backends/postgres/__init__.py

```diff
@@ -284,24 +288,60 @@ def list_tables(
         like
             A pattern to use for listing tables.
         schema
-            The schema to perform the list against.
+            [deprecated] The schema to perform the list against.
+        database
+            Database to list tables from. Default behavior is to show tables in
+            the current database.
 
-            ::: {.callout-warning}
-            ## `schema` refers to database hierarchy
+            ::: {.callout-note}
+            ## Ibis does not use the word `schema` to refer to database hierarchy.
 
-            The `schema` parameter does **not** refer to the column names and
-            types of `table`.
+            A collection of tables is referred to as a `database`.
+            A collection of `database` is referred to as a `catalog`.
+
+            These terms are mapped onto the corresponding features in each
+            backend (where available), regardless of whether the backend itself
+            uses the same terminology.
             :::
 
         """
+        if schema is not None:
+            # TODO: remove _warn_schema when the schema kwarg is removed
+            from ibis.util import _warn_schema
+
+            _warn_schema()
+
+        if schema is not None and database is not None:
+            raise ValueError(
+                "Using both the `schema` and `database` kwargs is not supported. "
+                "`schema` is deprecated and will be removed in Ibis 10.0"
+                "\nUse the `database` kwarg with one of the following patterns:"
+                '\ndatabase="database"'
+                '\ndatabase=("catalog", "database")'
+                '\ndatabase="catalog.database"',
+            )
```

Comment:
This is the odd duck out, as we had a `schema` kwarg in `list_tables` but no `database` kwarg, so this isn't a breaking change.